server/conf/i18n/messages

```diff
@@ -101,56 +101,98 @@ toast.programCompleted=Application was already completed.
 # TRUSTED INTERMEDIARY DASHBOARD PAGE - text when adding, editing, deleting, or searching for a client #
 #------------------------------------------------------------------------------------------------------#
 
+# Message that appears when a user attempts to delete an account and it fails.
 banner.acctDeleteError=This account cannot be deleted.
+# Reason included with the account delete error, which explains the account can't be deleted due to applications the client has open
 banner.acctDeleteErrorReason=This account cannot be deleted since it has open applications.
+# Message when information about a client, like their name or contact information is updated.
 banner.clientInfoUpdated=Client info has been successfully updated.
+# Message when a client account for a user is successfully deleted, with the name of the client substituted in
 banner.clientAcctDeleted=The client account for "{0}" has been successfully deleted.
+# Message when a user successfully creates a client account.
 banner.newClientCreated=New client successfully created.
 
+# Button that allows a user to add a client to the CiviForm system.
 button.addNewClient=Add new client
+# Button that brings the user back to the list of their clients.
 button.backToClientList=Back to client list
+# Button that allows the user to navigate back to the edit page after they click to delete a client and get an error.
 button.backToEditing=Back to editing
+# Button that allows the user to cancel the progress they make on adding or editing a client.
 button.cancel=Cancel
+# Button that allows the user to clear the search bar
```

Comment:
suggestion (non-blocking):  I think it might be clearer to say " ... to clear any search parameters they have already entered", since this search has multiple fields rather than a single search bar.

Reply:
Updated, thanks!!